core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -4800,6 +4819,11 @@ components:
               description: |
                 Whether the cached rejection of this intent is due to the intent already having been committed.
                 If so, see the /transaction/receipt endpoint for further information.
+            intent_already_committed_as:
+              $ref: "#/components/schemas/CommittedIntentMetadata"
+              description: |
+                Details of an already-committed transaction that had the same intent.
+                This field is present when `is_rejected_because_intent_already_committed` is true.
```

Comment:
_(note from author)_

There are a couple of ways to expose the "same intent-only vs same payload" distinction in the API.
The one picked here (`is_same_transaction: boolean`) is of course arbitrary, but that's what we discussed with David.